lib/redirect-logs.sh

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
 
+# Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2.0 License.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2015-Present Datadog, Inc.
```

Comment:
Then let's update to 2017. See https://github.com/DataDog/datadog-cloudfoundry-buildpack/pull/72/files#r362529642

Reply:
🤦‍♂